tests/shard_map_test.py

```diff
@@ -143,11 +141,11 @@ def fwd(a):
       return c
 
     c = fwd(a)
-    self.assertEqual(c.device_buffers[0].shape, (4, 8))
+    self.assertEqual(c.addressable_shards[0].shape, (4, 8))
 
   def test_matmul_reduce_scatter(self):
     mesh, a, b = create_inputs(P('z', 'y'), P('y', None))
-    assert a.device_buffers[0].shape == (4, 4)
+    assert a.addressable_shards[0].shape == (4, 4)
```

Comment:
Shouldn't this be .addressable_shards[0].data.shape?

Or better: .addressable_data(0).shape?